lib/presentation/widgets/loading_indicator.dart

```diff
@@ -1,12 +1,82 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:movna/assets.dart';
 
 /// The loading indicator used by the application
 class LoadingIndicator extends StatelessWidget {
   const LoadingIndicator({super.key});
 
   @override
   Widget build(BuildContext context) {
-    // TODO: use movna indicator https://github.com/MovnaTeam/movna/issues/24
-    return const CircularProgressIndicator();
+    return const IconSpinner();
+  }
+}
+
+class IconSpinner extends StatefulWidget {
+  const IconSpinner({
+    super.key,
+    this.width = 40,
+    this.duration = const Duration(seconds: 3),
+    this.curve = Curves.elasticInOut,
+  });
+
+  /// The width of the spinner, height is deduced from width.
+  final double width;
+
+  /// The duration of the animation.
+  final Duration duration;
+
+  /// The type of curve to be used.
+  final Curve curve;
+
+  @override
+  State<IconSpinner> createState() => _IconSpinnerState();
+}
+
+/// [AnimationController]s can be created with `vsync: this` because of
+/// [TickerProviderStateMixin].
```

Comment:
not sure this comment is needed

Reply:
Any resemblance with the animation example on the flutter website would be a pure coïncidence, not at all a copy-paste